### DISCUSSION TITLE
Is the --end-date parameter confusing?

### DISCUSSION THREAD

**Discussion by juftin** [juftin/camply#26] [Ideas]
Asking for any and all input. Since I built camply, I have defined the start/end date parameters as follows:
```
  --start-date START_DATE
                        (YYYY-MM-DD) Start of Search window. You will be
                        arriving this day.
  --end-date END_DATE   (YYYY-MM-DD) End of Search window. You will be leaving
                        the following day.
```

Now that I'm adding the `--nights` argument to search for consecutive nights at the same campsite I think this might be confusing. So I'm looking for feedback:

### 🚀) Final Camping Night - You will be leaving the following day

**Comment by juftin** [comment]
On second thought , I vote for 🎉 , `--end-date` should be the `Checkout Day - You will be leaving this day`. 

This will require a little bit of work to refactor but I think it makes more sense.

**Reply by kurbans** [reply]
To me it makes most sense as the checkout day since that will be the `end date` of the trip.

**Comment by juftin** [comment]
Update, the `end-date` change was implemented in v0.1.8
